packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -444,6 +444,7 @@ export default class Meeting extends StatelessWebexPlugin {
   annotation: any;
   conversationUrl: string;
   correlationId: string;
+  caCorrelationId: string;
```

Comment:
There is no need to have a separate new correlation id field - we should use existing `correlationId` field. Correlation id is used for identifying calls in logs, metrics, etc. so it's important that all the code always uses the same value for correlation id everywhere for a given meeting join. 
If you need the web app to specify the correlationId, then that should be done through an argument to Meeting.join() method - if the app specifies it, SDK should use it, otherwise SDK will generate a new random value (that's currently done by existing SDK code).

Reply:
yes, the original idea was that the correlationId of cross launch would overwrite the original correlationId, which would cause inconsistency in business logic. According to your suggestion, I will directly pass the correlationId through the meeting.join interface, which is a good way.